docker/consistency.dockerfile

```diff
@@ -2,13 +2,14 @@ FROM python:3.10-slim
 ENV PYTHONDONTWRITEBYTECODE=1
 USER root
 ARG REF=main
-RUN apt-get update && apt-get install -y time git pkg-config make git-lfs
+RUN apt-get update && apt-get install -y time git g++ pkg-config make git-lfs
```

Comment:
do we need g++ for pypi-kenlm?

Reply:
yes, didn't work without. same as in torch-tf-light